articles/cognitive-services/QnAMaker/limits.md

```diff
@@ -123,6 +123,9 @@ These represent the limits when Prebuilt API is used to *Generate response* or c
 > [!IMPORTANT]
 > Support for unstructured file/content and is available only in question answering.
 
+## Alterations limits
+[Alterations](https://docs.microsoft.com/ja-JP/rest/api/cognitiveservices/qnamaker/alterations/replace) do not allow these special characters: ',', '?', ':', ';', '\"', '\'', '(', ')', '{', '}', '[', ']', '-', '+', '.', '/', '!', '*', '-', '_', '@', '#'
```

Comment:
Link is currently hard-coded to Japanese which will break auto-localization of the link and everyone who clicks the link regardless of the language they are currently viewing the docs in will be taken to the Japanese version of the reference documentation. Removing ja-JP will fix this issue and the link will automatically take the user to the version of the reference docs that match the language of the page they are currently on.

```suggestion
[Alterations](https://docs.microsoft.com/rest/api/cognitiveservices/qnamaker/alterations/replace) do not allow these special characters: ',', '?', ':', ';', '\"', '\'', '(', ')', '{', '}', '[', ']', '-', '+', '.', '/', '!', '*', '-', '_', '@', '#'
```